zucchini-ui-backend/src/main/java/io/zucchiniui/backend/scenario/views/GroupedStepsListItemView.java

```diff
@@ -0,0 +1,32 @@
+package io.zucchiniui.backend.scenario.views;
+
+import io.zucchiniui.backend.scenario.domain.Step;
+import io.zucchiniui.backend.shared.domain.Location;
+
+import java.util.Set;
+
+public class GroupedStepsListItemView {
+
+    private Location stepDefinitionLocation;
+    private Set<Step> occurrences;
```

Comment:
A Occurence object would be better, with only properties info and status from the step.